link/kprobe.go

```diff
@@ -117,16 +126,16 @@ func pmuKprobe(symbol string, ret bool) (*perfEvent, error) {
 		return nil, err
 	}
 
-	// TODO: Parse the position of the bit from /sys/bus/event_source/devices/%s/format/retprobe.
-	config := 0
-	if ret {
```

Comment:
Why not just swap out the contents of the `if ret {` block? This leaves the existing structure in place, minimizes the diff, and keeps the `unix.PerfEventAttr` struct literal as it was before.